mazda_cx5_gt_2017.dbc

```diff
@@ -144,8 +144,8 @@ BO_ 579 CAM_LKAS: 8 XXX
  SG_ LDW : 23|1@0+ (1,0) [0|1] "" XXX
  SG_ LKAS_REQUEST : 3|12@0+ (1,-2048) [0|2048] "" XXX
  SG_ ERR_BIT_2 : 30|1@0+ (1,0) [0|1] "" XXX
- SG_ LKAS_ANGLE : 33|12@0+ (0.022,-45.06) [0|1] "" XXX
- SG_ BIT2 : 52|1@0+ (1,0) [0|1] "" XXX
+ SG_ ANGLE_ENABLED : 52|1@0+ (1,0) [0|1] "" XXX
+ SG_ STEERING_ANGLE : 33|12@0+ (1,-2048) [-2048|2047] "" XXX
```

Comment:
Looks like this should be signed

Reply:
Why? it is a 12 bit value [0:4095] wirh -2048 offset giving values in the range of [-2048:2047]. signed values are very rare in Mazda, offset is generally used.